pkg/genericapiserver/openapi_fixes.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package genericapiserver
+
+import (
+	"bytes"
+	"github.com/go-openapi/spec"
+	"strings"
+	"unicode"
+)
+
+// Functions in this file customize generated openAPI spec to make it a valid spec.
+// In an ideal world, these function should not exists, however current implementation of routes
+// (manly in api_installer.go) make it hard to fix all of these in a timely manner.
+
+type pathOperationID struct {
+	path string
+	id   string
+}
+
+var opFix = map[pathOperationID]string{
+	pathOperationID{"/api/v1/configmaps", "listNamespacedConfigMap"}:   "listAllNamespacesConfigMap",
```

Comment:
Can you add a comment why each of these require fixing? (Feel free to point to existing issues)